src/lightning/data/utilities/disk.py

```diff
@@ -0,0 +1,9 @@
+import subprocess
+
+
+def _get_available_amount_of_bytes() -> int:
+    """Returns the available amount of bytes on the machine."""
+    result = subprocess.check_output(["df", "-k", "/"])
+    result = [v for v in result.decode("utf-8").split("\n")[1].split(" ") if v != ""]
+    available = int(result[3])
+    return available
```

Comment:
I suggest to replace this with `shutil.disk_usage("/")`, or better even `shutil.disk_usage(cache_dir)`. You can then access `.total`, `.used` or `.free` from the return value.

Reply:
done !